examples/acoustic_example.py

```diff
@@ -71,7 +71,7 @@ def source(t, f0):
 receiver_coords[:, 2] = location[2]
 data.set_receiver_pos(receiver_coords)
 data.set_shape(nt, 101)
-Acoustic = Acoustic_cg(model, data)
+Acoustic = Acoustic_cg(model, data, auto_tune=True)
```

Comment:
Can this be removed from this PR and discussed somewhere else? Shouldn't be related anyway